src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -412,6 +410,35 @@ install_packages_in_root (RpmOstreeTreeComposeContext  *self,
       return FALSE;
   }
 
+  /* For unified core, we have a pkgcache repo. This may be auto-created under
+   * the workdir, or live explicitly in the dir for --cache.
+   */
+  glnx_autofd int host_rootfs_dfd = -1;
+  if (opt_ex_unified_core)
+    {
+      int pkgcache_dfd = self->cachedir_dfd != -1 ? self->cachedir_dfd : self->workdir_dfd;
+      self->pkgcache_repo = ostree_repo_create_at (pkgcache_dfd, "pkgcache-repo",
+                                                   OSTREE_REPO_MODE_BARE_USER, NULL,
+                                                   cancellable, error);
+      if (!self->pkgcache_repo)
+        return FALSE;
+      rpmostree_context_set_repos (self->corectx, self->repo, self->pkgcache_repo);
+      self->devino_cache = ostree_repo_devino_cache_new ();
+
+      /* Ensure that the imported packages are labeled with *a* policy if
+       * possible, even if it's not the final one. This helps avoid duplicating
+       * all of the content.
+       */
+      if (!glnx_opendirat (AT_FDCWD, "/", TRUE, &host_rootfs_dfd, error))
+        return FALSE;
+      g_autoptr(OstreeSePolicy) sepolicy = ostree_sepolicy_new_at (host_rootfs_dfd, cancellable, error);
+      if (!sepolicy)
```

Comment:
I should note this also means that `--ex-unified-core` requires having `selinux-policy-targeted` in the container/host.

Reply:
Let's just print a notice instead and keep going? It would make it easier to test from a container. :)
Anyway if caching, then we only pay that relabel cost once, right?

Reply:
One can install the policy in a container, that's what I've done.  We *could* stumble on but it means the relabel actually rewrites every object.  Seems easier to require people to install policy?

(That said of course this whole thing intersects with https://github.com/projectatomic/rpm-ostree/issues/971)